Source/Eos/Fuego.H

```diff
@@ -16,6 +16,38 @@ struct Fuego
 
   static std::string identifier() { return "Fuego"; }
 
+  AMREX_GPU_HOST_DEVICE
+  AMREX_FORCE_INLINE
+  static void RY2R(const amrex::Real RY[], amrex::Real& R)
+  {
+    R = 0.0;
+    for (int is = 0; is < NUM_SPECIES; is++) {
```

Comment:
using `is` is a slighly different style than `i` for for loops on species in this file. just noting this in case you want to do anything about it. I don't mind either way